pyg_lib/csrc/sampler/cpu/utils.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <torch/torch.h>
+
+#include "parallel_hashmap/phmap.h"
+
+template <typename scalar_t>
+inline torch::Tensor from_vector(const std::vector<scalar_t>& vec,
+                                 bool inplace = false) {
+  const auto size = (int64_t)vec.size();
+  const auto out = torch::from_blob((scalar_t*)vec.data(), {size},
+                                    c10::CppTypeToScalarType<scalar_t>::value);
+  return inplace ? out : out.clone();
+}
+
+template <typename key_t, typename scalar_t>
+inline c10::Dict<key_t, torch::Tensor> from_vector(
+    const phmap::flat_hash_map<key_t, std::vector<scalar_t>>& vec_dict,
+    bool inplace = false) {
+  c10::Dict<key_t, torch::Tensor> out_dict;
+  for (const auto& kv : vec_dict)
+    out_dict.insert(kv.first, from_vector<scalar_t>(kv.second, inplace));
+  return out_dict;
+}
+
+inline int64_t uniform_randint(int64_t low, int64_t high) {
```

Comment:
At best, we can use `cpu/random/rand_engine` for this and drop this functions altogether. See [here](https://github.com/pyg-team/pyg-lib/blob/master/test/csrc/random/test_rand_engine.cpp#L9) for an example.

Reply:
Done.